superset/views.py

```diff
@@ -1104,6 +1104,41 @@ def ping():
     return "OK"
 
 
+class KV(BaseSupersetView):
+
+    """Used for storing and retrieving key value pairs"""
+
+    @log_this
+    @expose("/store/", methods=['POST'])
+    def store(self):
+        value = request.form.get('data')
+        if not value:
+            return Response(
+                __("Error: data can not be empty"), status=404)
+        base_view = request.form.get('baseView')
+        if not base_view:
```

Comment:
let's not check on base_view and data validity, just try to fetch the value and return the exception message in case of error